generator/generator.ml

```diff
@@ -634,9 +650,6 @@ let check_functions () =
         if contains_uppercase n then
           failwithf "%s param/ret %s should not contain uppercase chars"
             name n;
-        if String.contains n '-' || String.contains n '_' then
-          failwithf "%s param/ret %s should not contain '-' or '_'"
-            name n;
```

Comment:
Why remove this?

Reply:
Removed check as it prevented creating methods with argument names including a "_"